angularjs/Grid/Editing.md

```diff
@@ -1,7 +1,7 @@
 ---
 Layout: Post
-title: Editing with Grid widget for the Syncfusion Essential AngularJS
-description: How to perform editing and configure edit time functionalities like edit type, edit time controls, etc
+title: Editing in Angular Grid Control | Syncfusion
+description: Leam here all about how to perform editing  in Syncfusion Angular Grid control its elements feautures and more
```

Comment:
Check this
![image](https://github.com/syncfusion-content/angularjs-docs/assets/90315799/ab9550c8-1c40-41ac-9370-af3c21cf2105)

Reply:
DONE